Expensify/App
# Initial Setup for wallet Transfer

### Details

Please review @marcaaron 

- [x] Do a simple PR to scaffold out some of the little changes first and add all the assets, consts, pages, etc:

    Add consts
    Add Routes
    Add blank pages for the new routes we are adding
    Translations
    API.js change
    Move payment methods list building logic into PaymentUtils without modifying the original logic and use in PaymentMethodList.


### Fixed Issues

$ #3922

### Tests | QA Steps
1. Go to the payments page.
2. Check that all payment methods are listed.
3. You should be able to click the PayPal method and navigate to the PayPal page.
4. Add Payment Method menu should work properly.

### Tested On


### Screenshots


#### Web


https://user-images.githubusercontent.com/24370807/145723085-ddec27cd-2e8b-4a7c-80e4-76bf04c5a923.mp4

#### Mobile Web


#### Desktop


#### iOS


#### Android
